weaknesses/MASVS-PRIVACY/MASWE-0117.md

```diff
@@ -0,0 +1,65 @@
+
+### Pre-installed Apps
+
+Pre-installed apps frequently come with excessive permissions that users cannot control or revoke, as they are often granted by default without explicit consent. This lack of control can result in continuous data collection and persistent privacy risks.
+
+### Third-party Libraries (SDKs)
+
+**Third-party libraries (SDKs)** further complicate permission management by inheriting app permissions and introducing privacy and security risks that are difficult to audit and control. Mobile permission models often fail to distinguish between permissions granted to an app and those assigned to third-party components, a challenge highlighted in the [IEEE research paper "Engineering Privacy in Smartphone Apps"](https://ieeexplore.ieee.org/document/9001128) (Section IV, _"Third-party content"_). Furthermore, third-party services behind these SDKs may continue accessing data collected over the network even after permissions are revoked or the app is deleted, creating long-term risks for user privacy.
+
+## Modes of Introduction
+
+- **Requesting Excessive Permissions**: Apps requesting more permissions than necessary for their core functionality.
+- **Lack of Use of Privacy-Friendly Alternatives**: Failing to use privacy-friendly alternatives to permissions that are less intrusive and provide users with more control over their data. For example, using coarse location instead of fine location, or using an image picker instead of requesting access to the camera and photo gallery.
+- **Lack of Proactive Permission Revocation**: Not automatically revoking app permissions that are no longer necessary, resulting in unnecessary data access over time.
+- **Inadequate Permission Explanations**: Failing to provide clear explanations for why each permission is required.
+
+## Impact
+
+- **Violation of User Privacy**: Users may have their personal data unnecessarily accessed by mobile apps, leading to potential misuse, identity theft, or surveillance.
+- **Loss of User Trust**: Users may lose trust in an app if it requests unnecessary permissions or does not allow them to revoke permissions that are no longer relevant. This can lead to negative reviews, lower user engagement, and reduced retention.
+- **Legal and Compliance Issues**: Apps that improperly manage permissions may face non-compliance with privacy regulations like GDPR or CCPA, which require data minimization and appropriate user control over data access, resulting in potential fines, legal action, or removal from app stores.
+- **Malicious Abuse:** Harmful apps can misuse permissions from privileged apps to record, track, or steal data without user consent.
+- **Data Breaches:** Once sensitive data leaves the app, it can be vulnerable to cyberattacks if it's not properly protected on remote endpoints, potentially leading to large-scale data leaks and reputational damage.
```

Comment:
```suggestion
- **Data Breaches:** Once sensitive data leaves the app, its safety can no longer be guaranteed and it may end up in large-scale data leaks.
```